README.md

```diff
@@ -1,12 +1,10 @@
 # Business Site
 
-Replace this readme with your own information about your project. 
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+Site with a hero image and a form.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+Due to health circunstances this week I didn't manage to finish the accordion, but I plan to do it as soon as I can (and also add a link to view it live).
```

Comment:
Ahh, I see. No worries, I'll do a code review on the existing code 👍 Hope you're feeling better 🌺